examples/community-detection.ipynb

```diff
@@ -0,0 +1,1148 @@
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Largest_CC"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "17942d04",
+   "metadata": {},
+   "source": [
+    "### Community Detection using Louvain\n",
+    "\n",
+    "We use the [Louvain](https://neo4j.com/docs/graph-data-science/2.4-preview/algorithms/louvain/) algorithm to detect communities in our subgraph and assign a louvainCommunityId to each community."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "def26464",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Louvain: 100%|██████████| 100.0/100 [00:12<00:00,  7.95%/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "mutateMillis                                                             0\n",
+       "nodePropertiesWritten                                                 3172\n",
+       "modularity                                                         0.58988\n",
+       "modularities             [0.4494089141198883, 0.5373675216145954, 0.555...\n",
+       "ranLevels                                                               10\n",
+       "communityCount                                                         300\n",
+       "communityDistribution    {'p99': 196, 'min': 1, 'max': 382, 'mean': 10....\n",
+       "postProcessingMillis                                                    22\n",
+       "preProcessingMillis                                                      1\n",
+       "computeMillis                                                        12974\n",
+       "configuration            {'maxIterations': 10, 'seedProperty': None, 'c...\n",
+       "Name: 0, dtype: object"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df2 = gds.louvain.mutate(Largest_CC, mutateProperty='louvainCommunityId')\n",
+    "df2"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "7563e824",
+   "metadata": {},
+   "source": [
+    "We get a modularity score of 0.5898 for our community detection algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "46969ec5",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.5898798012505129"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df2.modularity"
```

Comment:
we already see the modularity in the previous cell. How about dropping this cell?

Reply:
Done!